Cargo.toml

```diff
@@ -33,7 +33,8 @@ experimental = [
   "experimental-lazy-reader",
   "experimental-reader",
   "experimental-writer",
-  "experimental-streaming"
+  "experimental-streaming",
+  "experimental-serde"
```

Comment:
_extremely_ minor nitpick—please add a trailing comma so that when the next experimental feature is added, we don't don't have to add a comma to an unrelated feature.